frontends/mit-open/src/page-components/ManageListDialogs/ManageListDialogs.tsx

```diff
@@ -129,8 +130,10 @@ const UpsertLearningPathDialog = NiceModal.create(
     >["onSubmit"] = useCallback(
       async (values) => {
         if (resource?.id) {
+          // @ts-expect-error channel_url
           await updateList.mutateAsync({ ...values, id: resource.id })
         } else {
+          // @ts-expect-error channel_url
```

Comment:
If you make the change above  and the change in the serializer you can remove this and the @ts-expect-error  in line 133